modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -321,11 +321,22 @@
  * A <b>shearing</b> transformation, sometimes called a skew, effectively
  * rotates one axis so that the x and y axes are no longer perpendicular.
  * <p>
- * Multiple transformations may be applied to a node by specifying an ordered
- * chain of transforms.  The order in which the transforms are applied is
- * defined by the ObservableList specified in the {@link #getTransforms transforms} variable.
+ * Multiple transformations may be applied to a node. Custom transforms are applied using the
+ * {@link #getTransforms transforms} list. Predefined transforms are applied using the properties specified below.
+ * The matrices that represent the transforms are multiplied in this order:
+ * <ol>
+ * <li> Layout ({@link #layoutXProperty layoutX}, {@link #layoutYProperty layoutY}) and translate
+ * ({@link #translateXProperty translateX}, {@link #translateYProperty translateY}, {@link #translateZProperty translateZ})</li>
+ * <li> Rotate ({@link #rotateProperty rotate})</li>
+ * <li> Scale ({@link #scaleXProperty scaleX}, {@link #scaleYProperty scaleY}, {@link #scaleZProperty scaleZ})</li>
+ * <li> Transforms list ({@link #getTransforms transforms}) starting from element 0</li>
+ * </ol>
+ * The transforms are applied in the reverse order of the matrix multiplication outlined above: last element of the transforms list
+ * to 0th element, scale, rotate, and layout and translate. By applying the transforms in this order, the bounds in the local
+ * coordinates of the node are transformed to the bounds in the parent coordinate of the node (see the <b>Bounding Rectangles</b>
```

Comment:
Maybe hyperlink to the Bounding Rectangles section? (although as it is right below, it isn't as important).

Reply:
Originally I had "see the next section" but didn't want to depend on the order. I put the link anyway.